src/main/java/org/spongepowered/mod/mixin/core/event/MixinEventBus.java

```diff
@@ -48,7 +48,7 @@
     @Shadow
     private IEventExceptionHandler exceptionHandler;
 
-    @Overwrite
+    @Overwrite(constraints = "FML(1371 +10)")
```

Comment:
Yes but it's too scary a prospect to not mark it as a critical bit of infrastructure which should be monitored closely. Suggest a different tolerance?